test/e2e/util_test.go

```diff
@@ -79,6 +79,20 @@ func buildEchoPod(name, namespace string) *corev1.Pod {
 	}
 }
 
+// returns a SecurityContext with the minimum possible privileges that satisfy restricted pod security requirements
```

Comment:
*nit* I think it is a go doc standard to start with the function name
```suggestion
// generateUnprivilegedSecurityContext returns a SecurityContext with the minimum possible privileges that satisfy restricted pod security requirements
```